config/initializers/inflections.rb

```diff
@@ -12,5 +12,5 @@
 
 # These inflection rules are supported but not enabled by default:
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
+#   inflect.irregular 'party', 'parties'
```

Comment:
We thought that maybe an issue that we were running into was that party wasn't properly pluralizing, but it wasn't related so this code has been commented out. 

The issue was that our project is called viewing_party, so a module called ViewingParty was automatically created. This caused issues with our table name being called ViewParties. Resolved by renaming to Parties.